packages/next/types/index.d.ts

```diff
@@ -55,4 +56,15 @@ export type PageConfig = {
 
 export { NextPageContext, NextComponentType, NextApiResponse, NextApiRequest }
 
+export type Unstable_getStaticProps = (params: {
```

Comment:
Hmm, I'm not sure we want these exported yet. Can we keep them internal for now (`misc.d.ts`)?

Reply:
Moved them back to internal for now